hadoop-hdds/docs/content/beyond/Containers.md

```diff
@@ -46,38 +47,41 @@ Ozone artifact contains example docker-compose directories to make it easier to
 
 From distribution:
 
-```
+```bash
 cd compose/ozone
 docker-compose up -d
 ```
 
-After a local build
+After a local build:
 
-```
+```bash
 cd  hadoop-ozone/dist/target/ozone-*/compose
 docker-compose up -d
 ```
 
 These environments are very important tools to start different type of Ozone clusters at any time.
 
-To be sure that the compose files are up-to-date, we also provide acceptance test suites which start the cluster and check the basic behaviour.
+To be sure that the compose files are up-to-date, we also provide acceptance test suites which start
+the cluster and check the basic behaviour.
```

Comment:
typo behaviour

Reply:
Will fix it.
Behavior is the preferred spelling in American English. Behaviour is preferred everywhere else. Other than the spelling, there is no difference between the two words. 😃